modules/preview-react/breadcrumbs/lib/Breadcrumbs/Dropdown/Button.tsx

```diff
@@ -36,14 +35,12 @@ export const DropdownButton = createComponent(IconButton)({
     Element
   ) => {
     const hasPlainVariant = variant === 'plain';
-    // gives the plain icon variant a little extra space between the chevron separators
-    const customStyles = css(hasPlainVariant ? {margin: '0 1px'} : {});
 
     return (
       <Element
         ref={ref}
         type="button"
-        css={customStyles}
+        style={hasPlainVariant ? {margin: '0 1px'} : {}}
```

Comment:
Any ideas how to tackle this? It's an icon button underneath, can I use Box here?